hospital/src/test/java/by/emel/anton/facade/doctor/DoctorFacadeImplTest.java

```diff
@@ -0,0 +1,122 @@
+package by.emel.anton.facade.doctor;
+
+import by.emel.anton.facade.converter.DoctorConverter;
+import by.emel.anton.facade.therapy.RequestTherapyDTO;
+import by.emel.anton.model.beans.users.doctors.Doctor;
+import by.emel.anton.model.beans.users.patients.Patient;
+import by.emel.anton.model.dao.exceptions.UserDaoException;
+import by.emel.anton.service.UserService;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Spy;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import javax.servlet.http.HttpSession;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.*;
+
+
+@RunWith(MockitoJUnitRunner.class)
+public class DoctorFacadeImplTest {
+    @InjectMocks
+    private DoctorFacadeImpl doctorFacade;
+    @Mock
+    private UserService userService;
+    @Spy
+    private DoctorConverter converter;
+    @Mock
+    private HttpSession httpSession;
+
+    private static Doctor doctor;
```

Comment:
Do not use static fields and `@BeforeClass`. 
Use instance fields and `@BeforeEach` instead.
Update Spring Boot version to the latest one and refactor to use JUnit 5 framework